modules/apps/object/object-js-components-web/src/main/resources/META-INF/resources/components/BuilderScreen/BuilderScreen.tsx

```diff
@@ -51,113 +51,123 @@ export function BuilderScreen({
 
 	return (
 		<>
-			<ManagementToolbar.Container>
-				<ManagementToolbar.ItemList expand>
-					<ManagementToolbarSearch
-						query={query}
-						setQuery={setQuery}
-					/>
-
-					<ManagementToolbar.Item>
-						<ClayButtonWithIcon
-							aria-label={Liferay.Language.get('add')}
-							className="nav-btn nav-btn-monospaced"
-							onClick={openModal}
-							symbol="plus"
-						/>
-					</ManagementToolbar.Item>
-				</ManagementToolbar.ItemList>
-			</ManagementToolbar.Container>
-
 			{tableItems.length ? (
 				<ClayList>
-					{tableItems.map((viewColumn, index) => (
-						<React.Fragment key={viewColumn?.objectFieldName}>
-							{index === 0 && (
-								<ClayList.Item flex>
-									<ClayList.ItemField
-										className={classNames(
-											'lfr-object__object-builder-screen-first-column',
-											!hasDragAndDrop &&
-												'lfr-object__object-builder-screen-first-column--not-draggable'
-										)}
-										expand
-									>
-										{firstColumnHeader}
-									</ClayList.ItemField>
+					<ClayList.Item className="lfr-object__object-builder-screen-list">
+						<ClayList.Item className="lfr-object__object-builder-screen-list-management-bar">
+							<ManagementToolbar.Container>
+								<ManagementToolbar.ItemList expand>
+									<ManagementToolbarSearch
+										query={query}
+										setQuery={setQuery}
+									/>
 
-									<ClayList.ItemField
-										className={classNames(
-											'lfr-object__object-builder-screen-second-column',
-											!hasDragAndDrop &&
-												'lfr-object__object-builder-screen-second-column--not-draggable'
-										)}
-										expand
-									>
-										<ClayList.ItemField>
-											{secondColumnHeader}
+									<ManagementToolbar.Item>
+										<ClayButtonWithIcon
+											aria-label={Liferay.Language.get(
+												'add'
+											)}
+											className="nav-btn nav-btn-monospaced"
```

Comment:
Hey @larissacribeiro, you define the `nav-btn` here and after use `!important` to overwrite a style from this class. Could you check the effot to create your class to styled this buttom? To avoid use `!important`

Reply:
Hey @igor-franca, thanks, just did that!

Reply:
Hey @igor-franca, thanks, just did that!